bin/dfx-sns-demo-mksns

```diff
@@ -15,12 +15,13 @@ source "$SOURCE_DIR/clap.bash"
 # Define options
 clap.define short=n long=network desc="The dfx network to use" variable=DFX_NETWORK default="local"
 clap.define long=confirmation_text desc="A text that needs to be confirmed by the participants" variable=CONFIRMATION_TEXT default=""
+clap.define short=i long=config_index desc="The start value for the dfx-sns-config-random --config_index parameter" variable=CONFIG_INDEX default="0"
```

Comment:
What do you mean by "start" in "start value"?

Reply:
Thanks, updated.